test/test_torch.py

```diff
@@ -8572,6 +8572,9 @@ def test_Size(self):
         self.assertIsInstance(x[:-1], torch.Size)
         self.assertIsInstance(x + x, torch.Size)
 
+        # self.assertIsInstance((4, 5) + x, torch.Size)
+        # self.assertIsInstance((4, 5, 6) + x, torch.Size)
```

Comment:
I guess you meant to have these not as comments?

Reply:
Hi @albanD! Thanks for looking into it.
Yes, but the tests are still failing. I added `THPSize_radd` referring to [this PR](https://github.com/pytorch/pytorch/commit/b9ece39685742a12c7bd599b09fde240e1087ad3)), which returns `torch.Size`. However, it still calls `__add__`, and `__radd__` is never triggered. I'm exploring other possible solutions. Could you please confirm if the PR is on the right track?

Reply:
Gentle ping, @albanD!

Reply:
Is the reason `__add__` is called because the tuple+Size addition succeeds by converting the Size to a tuple?
If so, there isn't much you can do here :/